frontend/packages/ceph-storage-plugin/src/components/ocs-install/create-form.tsx

```diff
@@ -34,7 +34,7 @@ export const CreateOCSServiceForm: React.FC<CreateOCSServiceFormProps> = ({
           title="A bucket will be created to provide the OCS Service."
           isInline
         />
-        <p className="co-legend co-required ceph-ocs-desc__legend">
+        <p className="co-legend co-required ceph-ocs-co-required">
```

Comment:
Why can't we use 'co-required'..?

Reply:
`co-required` style is already used. See it carefully. Few css styles have been overwritten as the `co-required` styles were written with assumption of single line text.

Reply:
Single line text assumption fits everywhere except this page.